src/cbmc_starter_kit/etc/cbmc-starter-kit-bash-completion.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Please install in `etc/bash_completion.d/` for it to be automatically found on most systems when bash completion is enabled.

Reply:
```bash
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0
```

could be added

Reply:
Added MIT-0 license to all source files.  Moved bash completion to etc/bash_completion.d/cbmc-starter-kit.sh, but as discussed earlier, we can install under brew but there is no os-independent way to install under pip.